pkg/azureutils/azure_disk_utils.go

```diff
@@ -1165,52 +1125,49 @@ func UpdateCRIWithRetry(ctx context.Context, informerFactory azdiskinformers.Sha
 			w.Logger().V(5).Info("Skip update. No update needed.")
 			return nil
 		}
-
+		var updateStatusError error
```

Comment:
There's no need for `updateStatusError` and `updateError` now and these changes can be reverted.

Reply:
ah yes. I just wished to save some lines. But it should return err immediately if UpdateStatus() has err.